administrator/components/com_admin/sql/updates/mysql/4.2.0-2022-05-15.sql

```diff
@@ -0,0 +1,57 @@
+--
+-- Create the new table for MFA
+--
+CREATE TABLE IF NOT EXISTS `#__user_mfa` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `user_id` int unsigned NOT NULL,
+  `title` varchar(255) NOT NULL DEFAULT '',
+  `method` varchar(100) NOT NULL,
+  `default` tinyint NOT NULL DEFAULT 0,
+  `options` mediumtext NOT NULL,
+  `created_on` datetime NOT NULL,
+  `last_used` datetime,
+  PRIMARY KEY (`id`),
+  KEY `idx_user_id` (`user_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 DEFAULT COLLATE=utf8mb4_unicode_ci COMMENT='Multi-factor Authentication settings';
+
+--
+-- Remove obsolete postinstallation message
+--
+DELETE FROM `#__postinstall_messages` WHERE `condition_file` = 'site://plugins/multifactorauth/totp/postinstall/actions.php';
+
+--
+-- Add new MFA plugins
+--
+INSERT INTO `#__extensions` (`package_id`, `name`, `type`, `element`, `folder`, `client_id`, `enabled`, `access`, `protected`, `locked`, `manifest_cache`, `params`, `custom_data`, `ordering`, `state`) VALUES
+(0, 'plg_multifactorauth_totp', 'plugin', 'totp', 'multifactorauth', 0, 0, 1, 0, 1, '', '', '', 1, 0),
+(0, 'plg_multifactorauth_yubikey', 'plugin', 'yubikey', 'multifactorauth', 0, 0, 1, 0, 1, '', '', '', 2, 0),
+(0, 'plg_multifactorauth_webauthn', 'plugin', 'webauthn', 'multifactorauth', 0, 0, 1, 0, 1, '', '', '', 3, 0),
+(0, 'plg_multifactorauth_email', 'plugin', 'email', 'multifactorauth', 0, 0, 1, 0, 1, '', '', '', 4, 0),
+(0, 'plg_multifactorauth_fixed', 'plugin', 'fixed', 'multifactorauth', 0, 0, 1, 0, 1, '', '', '', 5, 0);
+
+--
+-- Update MFA plugins' publish status
+--
+UPDATE `#__extensions` AS `a`
+	INNER JOIN `#__extensions` AS `b` on `a`.`element` = `b`.`element`
+SET `a`.enabled = `b`.enabled
+WHERE `a`.folder = 'multifactorauth'
+	AND `a`.folder = 'twofactorauth';
```

Comment:
```suggestion
WHERE `a`.folder = 'multifactorauth'
	AND `b`.folder = 'twofactorauth';
```
Old (twofactorauth) plugin is not enabled after update